Buildings/Fluid/HeatExchangers/DXCoils/BaseClasses/CoilDefrostCapacity.mo

```diff
@@ -0,0 +1,257 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses;
+block CoilDefrostCapacity
+  "Calculates defrost curve value at given temperature and mass flow rate"
+  extends Modelica.Blocks.Icons.Block;
+
+  replaceable package MediumA = Modelica.Media.Interfaces.PartialMedium
+    "Fluid medium package";
+
+  parameter Real tDefRun(
+    final unit="1",
+    displayUnit="1") = 0.5
+    "Time period for which defrost cycle is run"
+    annotation(Dialog(enable=defTri == Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses.Types.DefrostTimeMethods.timed));
+
+  parameter Modelica.Units.SI.Power QDefResCap = defCur.QDefResCap
+    "Capacity of resistive defrost element";
+
+  parameter
+    Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses.Types.DefrostTimeMethods
+    defTri=Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses.Types.DefrostTimeMethods.timed
+    "Type of method to trigger the defrost cycle";
+
+  parameter
+    Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses.Types.DefrostOperation
+    defOpe=Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses.Types.DefrostOperation.resistive
+    "Type of defrost method";
+
+  replaceable parameter
+    Buildings.Fluid.HeatExchangers.DXCoils.AirSource.Data.Generic.BaseClasses.Defrost
+    defCur constrainedby
+    Buildings.Fluid.HeatExchangers.DXCoils.AirSource.Data.Generic.BaseClasses.Defrost
+    "Defrost curve parameter record";
+
+  Modelica.Blocks.Interfaces.RealInput tDefFra(final unit="1", displayUnit="1")
+    "Defrost time period fraction" annotation (Placement(transformation(extent={
+            {-120,90},{-100,110}}), iconTransformation(extent={{-120,100},{-100,
+            120}})));
+
+  Modelica.Blocks.Interfaces.RealInput heaCapMul(
+    final unit="1",
+    displayUnit="1")
```

Comment:
If it has `unit="1"` there is no need to add `displayUnit="1"`, please remove it. If not specified it will be displayed using `unit`